static/resources/yaml/datadog-agent-npm.yaml

```diff
@@ -44,7 +41,11 @@ data:
       bpf_debug: false
       enable_tcp_queue_length: false
       enable_oom_kill: false
-      collect_dns_stats: true
+      collect_dns_stats: false
```

Comment:
This change of `collect_dns_stats` from `true` to `false` is surprising, but I’m OK to stick with the default value here.